raiden/network/transport.py

```diff
@@ -106,6 +106,14 @@ def send(self, sender, host_port, bytes_):
 
     def stop(self):
         self.server.stop()
+        # Calling `.close()` on a gevent socket doesn't actually close the underlying os socket
+        # so we do that ourselves here.
+        # See: https://github.com/gevent/gevent/blob/master/src/gevent/_socket2.py#L208
+        # and: https://groups.google.com/forum/#!msg/gevent/Ro8lRra3nH0/ZENgEXrr6M0J
+        try:
+            self.server._socket.close()
+        except Exception:
```

Comment:
why the catch-all? (This may ignore gevent's exception used to kill greenlets)

Reply:
@ulope Ideas? I'm not sure about this one.

Reply:
As far as I can tell it only throws [`socket.error`](https://github.com/certik/python-2.7/blob/master/Lib/socket.py#L167) exceptions.